webplugin/js/app/conversation/gen-ai-service.js

```diff
@@ -0,0 +1,43 @@
+class GenAiService {
+    constructor() {
+        this.currentElement = null;
+        this.textMsgDiv = null;
+    }
+
+    addTokenizeMsg = (...args) => {
+        const [msg, className, $textMessage] = args;
+
+        if (!this.currentElement) {
```

Comment:
is it possible that the currentElement exists but for a different message key, and not for the msg.key which is passed?

Reply:
no, this case will not happen because I'm resetting the currentElement and textMsgDiv values when the user sends the message.